recipe/meta.yaml

```diff
@@ -132,6 +133,7 @@ outputs:
         - libtiff
         - libblas
         - liblapack
+        - tzdata
```

Comment:
If we do the `pin_compatible` below, we should fix this to a lower version (e.g., the one from source package and/or what the added test uses).
Otherwise it's unnecessarily restrictive, IMHO.